docs/S03-smart-contracts/M4-design-patterns/L4-oracles/index.md

```diff
@@ -8,9 +8,9 @@ Additionally, what if we'd like to interact with external computation in the out
 
 Such data and external computation exists but they are beyond the _trust boundary_ established by our blockchain protocols.
 
-One way blockchain developers have tried to bridge the gap is through what are called **off-chain [oracles.](https://en.wikipedia.org/wiki/Blockchain_oracle){target=_blank}** Off-chain oracles are agents that find and verify real-world information and submit them to the blockchain to be used by smart contracts. They can trigger smart contract executions when the data is obtained or predefined conditions are meet (e.g. time, weather, tracking, payments). Off-chain oracles are provided by organizations like [Provable Things,](https://provable.xyz/){target=_blank} [Chainlink Labs,](https://chain.link/){target=_blank} and more.
+One way blockchain developers have tried to bridge the gap is through what are called **off-chain [oracles.](https://en.wikipedia.org/wiki/Blockchain_oracle)** Off-chain oracles are agents that find and verify real-world information and submit them to the blockchain to be used by smart contracts. They can trigger smart contract executions when the data is obtained or predefined conditions are meet (e.g. time, weather, tracking, payments). Off-chain oracles are provided by organizations like [Provable Things,](https://provable.xyz/{target=_blank}) [Chainlink Labs,](https://chain.link/{target=_blank}) and more.
```

Comment:
These ones are still broken! Might be easier to start fresh?

Reply:
Actually curious about the `{target=_blank}`. Is this necessary for the course on the consensys site?  Because the links work fine without that on the preview in GitHub.
I do see this issue in why they weren't working though, I included the `{target=_blank}` in the hyper link

Reply:
It is necessary, you can see the explanation here at the beginning of CONTRIBUTING.md:

https://github.com/ConsenSys-Academy/Blockchain-Developer-Bootcamp/blob/staging/CONTRIBUTING.md#background